.github/workflows/build_docker_images_for_ci_centos.yml

```diff
@@ -0,0 +1,35 @@
+name: Build Docker Images for CI
```

Comment:
This is Centos, is the title wrong?

Reply:
Nop, as I said a also updated the other workflows